From ed05ce74cfb9151ea5218da0f8b9eccb70c00f70 Mon Sep 17 00:00:00 2001
From: Nicola Sella <nsella@redhat.com>
Date: Thu, 11 Nov 2021 13:48:39 +0100
Subject: [PATCH] Update documentation for adding specific version (RhBug:2013332)

=changelog=
msg: [versionlock] update documentation for adding specifi version
type: bugfix
resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2013332
---
 doc/versionlock.rst | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/doc/versionlock.rst b/doc/versionlock.rst
index 061ce80..1ac7196 100644
--- a/doc/versionlock.rst
+++ b/doc/versionlock.rst
@@ -97,6 +97,10 @@ Subcommands
             Adding versionlock on: mutt-5:1.11.4-1.fc30.*
             Adding versionlock on: mutt-5:1.12.1-3.fc30.*
 
+    .. note:: Be careful when adding specific versions
+
+        If you add a package specifying a version with ``dnf versionlock mutt-5:1.11.4-1.fc30.x86_64`` then, if you run ``dnf versionlock add mutt``
+        versionlock will not add ``mutt-5:1.12.1-3.fc30.x86_64``.
 
 ``dnf versionlock exclude <package-name-spec>``
     Add an exclude (within  versionlock) for the available packages matching the spec. It means that
--
libgit2 1.1.0

